backend/src/v2/driver/driver.go

```diff
@@ -1078,12 +1079,6 @@ func provisionOutputs(pipelineRoot, taskName string, outputsSpec *pipelinespec.C
 	return outputs
 }
 
-func generateOutputURI(root, artifactName string, taskName string) string {
-	// we cannot path.Join(root, taskName, artifactName), because root
```

Comment:
Since now you new function is similar to the old `generateOutputURI` function, it make sense to bring these comments to the changes you have at line 1065 or just update the `generateOutputURI` function.